frontend/src/routes/Infrastructure/Clusters/ManagedClusters/components/OnboardingModal.tsx

```diff
@@ -89,7 +91,7 @@ export function OnboardingModal(props: IOnboardingModalProps) {
                             }}
                         >
                             {t('Learn more about Red Hat Advanced Cluster Management for Kubernetes')}{' '}
-                            <ExternalLinkAltIcon />
+                            <ExternalLinkAltIcon style={{ verticalAlign: '-0.125em', padding: '2px' }} />
```

Comment:
The space is still underlined here and should be removed.
![image](https://user-images.githubusercontent.com/42188127/211110117-0bbff45a-cf92-4392-bb3f-c805fb14f443.png)

But it would be even better if instead of hard-coding padding, you use the [inline link variant of a button](https://www.patternfly.org/v4/components/button#variant-examples), along with the `icon` and `iconPosition` props.